docs/CICD_BEST_PRACTICES.md

```
@@ -15,29 +16,33 @@ This document outlines CI/CD best practices adopted in the Mystira infrastructur
 **Concurrency Controls (Implemented)**
+
 ```yaml
 concurrency:
   group: terraform-${{ inputs.environment }}-${{ inputs.product }}
-  cancel-in-progress: false  # Don't cancel infrastructure operations
+  cancel-in-progress: false # Don't cancel infrastructure operations
 ```
 
 ### Infrastructure as Code
 
 **Terragrunt Product-Based Structure (Implemented)**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)